packages/terra-brand-footer/src/BrandFooter.scss

```diff
@@ -0,0 +1,54 @@
+:local {
+  .brand-footer {
+    margin: none;
+  }
+
+  .nav {
+    background-color: var(--terra-brand-footer-nav-background-color, #f4f4f4);
+    color: var(--terra-brand-footer-nav-color, #64696c);
+    display: block;
+    padding: 0.7142857142857143rem 1.4285714285714286rem;
+  }
+
+  .menu {
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .list-item {
+    display: inline;
+  }
+
+
+  .list-item:first-child::before {
+    content: '';
+    padding: 0;
+  }
+
+  .list-item::before {
+    content: '/';
```

Comment:
Why are we doing this?

Reply:
Also, is this internationalized well? Not all locales have the same separators.

Reply:
> Why are we doing this?

It is a design aesthetic from the legacy brand footer.

> Also, is this internationalized well? 

I don't think we need to internationalize this separator as it is a design aesthetic. IMO, it's different than a seperator that would be used in a number like a comma, or a separator in a date, like a dash.

Reply:
@emilyrohrbough Can you take a look at the screenshot down [here](https://github.com/cerner/terra-framework/issues/143).